src/NuGet.Core/NuGet.ProjectModel/FileFormatException.cs

```diff
@@ -10,6 +10,8 @@ namespace NuGet.ProjectModel
 {
     public class FileFormatException : Exception
     {
+        static internal string SurfaceMessage = "SurfaceMessage";
```

Comment:
Can this be `const` or does it need to be accessible without an exception instance?
Otherwise, I'd suggest readonly:

```suggestion
        static internal readonly string SurfaceMessage = "SurfaceMessage";
```